src/pgn.ts

```diff
@@ -135,5 +135,6 @@ function makeMetadata(headers: Headers, lichess: Lichess): Metadata {
     isLichess: !!(lichess && site?.startsWith(lichess)),
     timeControl,
     orientation: orientation === 'white' || orientation === 'black' ? orientation : undefined,
+    name: event ?? '?',
```

Comment:
in a broadcast, all games will have the same event. Probably better to try with the player names first.

Reply:
I think it would be useful to display an index (+1) in the select options, to always allow distinguishing the games, whether they have distinct names or not.